app/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/views/SingleAppealConfirmationView.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.ViewUtils.titleBuilder
+@import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html.Layout
+@import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html.components
+@import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.models.CurrentUserRequestWithAnswers
+
+
+@this(layout: Layout,
+        govukButton : GovukButton,
+        formHelper: FormWithCSRF,
+        p: components.P,
+        h1: components.H1,
+        caption: components.PenaltyCaption
+)
+
+@(penalty: Boolean, amount: BigDecimal)(implicit request: CurrentUserRequestWithAnswers[_], messages: Messages)
```

Comment:
Could you give this boolean a more informative name? The standards for a boolean are usually `isSomething` or `hasSomething`. I think in this case you could call it `isLPP1`